.github/workflows/release.yml

```diff
@@ -0,0 +1,80 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*.*.*'
```

Comment:
@moikot -- would you update this based on the comment here: https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/pull/21#issuecomment-747257853

Reply:
Yes, I'll update this part and the script needs to be updated a bit. I'll do it and validate the behavior.

Reply:
I'm planning to keep only this:
```yaml
on:
  push:
    tags:
      - 'v*.*.*'
````
Since neither branches nor pull-request handling is needed for forks I guess. Or I can keep them and suppress publishing, but in this case, the pipeline will be triggered in `kubernetes-sigs/nfs-subdir-external-provisioner` (without pushing though) @kmova what would you prefer?

Reply:
Pushing only for tags looks clean. 

We can add comments in this release yaml - to let people know how they can enable on the branches or provide an example in the readme for github multi-arch builds.

Reply:
Ok, then I leave the logic for branches and PRs but set the trigger for tags only.

Reply:
Done